git-*: convert options to longform and minor fixes (#13441)
tldr-pages/tldr: pages/common/git-log.md

- Show the history of a particular file or directory, including differences:

> `git log {{--patch|-p}} {{path/to/file_or_directory}}`

Reviewer: ```suggestion
`git log {{--patch|-p|-u}} {{path/to/file_or_directory}}`
```

`git log --oneline --decorate --all --graph`

> - Show only commits whose messages include a given string (case-insensitively):

Reviewer: ```suggestion
- Show only commits with messages that include a specific string, ignoring case:
```
